bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/BitcoindRpcClient.scala

```diff
@@ -129,6 +130,13 @@ class BitcoindRpcClient(override val instance: BitcoindInstance)(implicit
   override def epochSecondToBlockHeight(time: Long): Future[Int] =
     Future.successful(0)
 
+  override def getMedianTimePast(): Future[Long] = {
+    for {
+      hash <- getBestBlockHash
+      header <- getBlockHeader(hash)
+    } yield header.mediantime.toLong
```

Comment:
instead of this you can call `getblockchaininfo` and it has a `mediantime` field